hardware_accelerators/about-hardware-accelerators.adoc

```diff
@@ -0,0 +1,50 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="about-hardware-accelerators"]
+= About hardware accelerators
+include::_attributes/common-attributes.adoc[]
+:context: gpu-accelerators
+
+toc::[]
+
+Specialized hardware accelerators play a key role in the emerging generative artificial intelligence and machine learning (AI/ML) industry. Specifically, hardware accelerators are essential to the training and serving of large language and other foundational models that power this new technology. It allows data scientists, data engineers, ML engineers, and developers to take advantage of the specialized hardware acceleration for data-intensive transformations and model development and serving. Much of that ecosystem is open source with a number of contributing vendors and open source foundations.
+
+Red{nbsp}Hat OpenShift provides support for cards and peripheral hardware that add processing units that comprise hardware accelerators:
+
+* Graphical processing units (GPUs)
+* Tensor processing units (TPUs) 
+* Neural processing units (NPUs)
+* Data processing units (DPUs)
+
+Specialized hardware accelerators provide a rich set of benefits for AI/ML development:
+
+* One platform for all. Allows for collaborative environments for developers, data engineers, data scientists, and DevOps.
+* Extended capabilities with Operators. Operators allow for bringing AI/ML capabilities to OpenShift.
+* Hybrid-cloud support. On-premise support for model development, delivery, and deployment. 
+* Support for AI/ML workloads. Model testing, iteration, integration, promotion, and serving into production as services.
+
+Red{nbsp}Hat provides an optimized platform to enable these specialized hardware accelerators in our RHEL and OpenShift platforms at the Linux (kernel and userspace) and Kubernetes layers. To do this, Red{nbsp}Hat combines the proven capabilities of Red{nbsp}Hat OpenShift AI and Red{nbsp}Hat OpenShift in a single enterprise-ready AI application platform. 
+
+// GRAPHIC HERE
+
+Red{nbsp}Hat OpenShift AI is an open hybrid AI/ML platform built on Red{nbsp}Hat OpenShift that enables enterprises to create and deliver AI-enabled applications securely and at scale across hybrid clouds. It enables data acquisition and preparation, model training and fine-tuning, model serving and model monitoring, and hardware acceleration. 
+
+// Link to RHOAI landing page
+
+Specialized hardware accelerators are designed to work within air gapped (disconnected) environments. This is because of customer requirements to maintain a secure environment for development and testing.  
+
+Hardware Operators use the operating framework of a Kubernetes cluster to enable the required accelerator resources. A device plugin is also provided that can be deployed manually or as a DaemonSet. This plugin enables the registration of the GPU in the cluster.   
+
+== Supported vendors
+
+Red{nbsp}Hat Openshift supports hardware accelerators provided by the following vendors:
+
+* AMD
+* NVIDIA
+* Intel
+
```

Comment:
Do we support these for OpenShift?  I don't think so.  We provide accelerator tools.  The partners are responsible for their operators which integrate with our accelerator tools.  

RHEL AI is a different topic.  In this case Red Hat does support the partner accelerator specific images and content.

@egallen thoughts?

Reply:
Yes, better to remove the word support.